### DISCUSSION TITLE
Updated TW -> renders unexpected :not(#/#) results

### DISCUSSION THREAD

**Discussion by StudioRATATA** [tailwindlabs/tailwindcss#9286] [Help]
Hey all,

Forgive my ignorance but I recently updated my TW dependency. After getting unexpected css renders with a messed up layout I saw all css rules got an aditional `:not(#/#)` selector rendered to them. Like:

```
body:not(#\#):not(#\#):not(#\#) ::selection, html:not(#\#):not(#\#):not(#\#) ::selection{...}
```

Is this a bug or a feature? :) :) I don't know if this is something new and I have to rethink my workflow or maybe could be disabled or if it's a bug.

Any help would be much appreciated.

**Comment by wongjn** [comment]
Looks like a bug to me! How have you got your CSS build pipeline like?

**Reply by StudioRATATA** [reply]
Thanks, this pointed me out I don't need to check the docs for some new function :) I've narrowed it down to the use of `@apply my-class;`. Previous versions cat me some slack for not encapsulating this in `@layer components {...}`. But this version gives me the current error. 

I'm using TW in a dependency for a WP theme framework. The dependency is github.com/roots/bud/ (This regulates TW)
